On branch edburns-msft-dd-1618567-fileset-scaninterval-clarify-does-not-scan-for-changes-in-file

edburns opened the pull request:
modified:   dev/com.ibm.ws.kernel.metatype.helper/resources/OSGI-INF/l10n/fileset.properties
@nottycode said:

> so I've verified with someone on our dev team that the change would be valid, so if you submit a PR it wouldn't be rejected as invalid. They might suggest alternative wording, but if you want to do the PR I'm happy to work with them to get it merged.

Signed-off-by: Ed Burns <edburns@microsoft.com>

edburns committed:
On branch edburns-msft-dd-1618567-fileset-scaninterval-clarify-does-not-scan-for-changes-in-file
modified:   dev/com.ibm.ws.kernel.metatype.helper/resources/OSGI-INF/l10n/fileset.properties
@nottycode said:

> so I've verified with someone on our dev team that the change would be valid, so if you submit a PR it wouldn't be rejected as invalid. They might suggest alternative wording, but if you want to do the PR I'm happy to work with them to get it merged.

Signed-off-by: Ed Burns <edburns@microsoft.com>

edburns commented:
> Please describe in a separate comment how you tested your changes.

I did not.

hiltoncv commented on dev/com.ibm.ws.kernel.metatype.helper/resources/OSGI-INF/l10n/fileset.properties:
```diff
@@ -31,4 +31,4 @@ fileset.attr.excludes.name=Excludes pattern
 #monitor
 fileset.attr.scan.name=Scan interval
-fileset.attr.scan.desc=Scanning interval to check the fileset for changes as a long with a time unit suffix h-hour, m-minute, s-second, ms-millisecond (e.g. 2ms or 5s). Disabled (scanInterval=0) by default.
+fileset.attr.scan.desc=Scanning interval to check the fileset for changes as a long with a time unit suffix h-hour, m-minute, s-second, ms-millisecond (e.g. 2ms or 5s). Disabled (scanInterval=0) by default. Note: the contents of the individual files are not scanned. Rather, the fileset itself is scanned to determine if files are added or removed from the set.
```
Replace"fileset" with "file set".
Consider the text as
"Scanning interval to check the file set for changes as a long with a time unit suffix h-hour, m-minute, s-second, ms-millisecond (e.g. 2ms or 5s). Disabled (scanInterval=0) by default. Note: the contents of the individual files are not scanned. Rather, the fileset itself is scanned to determine if files are added or removed from the set."

hiltoncv reviewed (commented):
What is the name of the attribute that enables file scanning?

chirp1 commented on dev/com.ibm.ws.kernel.metatype.helper/resources/OSGI-INF/l10n/fileset.properties:
```diff
@@ -31,4 +31,4 @@ fileset.attr.excludes.name=Excludes pattern
 #monitor
 fileset.attr.scan.name=Scan interval
-fileset.attr.scan.desc=Scanning interval to check the fileset for changes as a long with a time unit suffix h-hour, m-minute, s-second, ms-millisecond (e.g. 2ms or 5s). Disabled (scanInterval=0) by default.
+fileset.attr.scan.desc=Scanning interval to check the fileset for changes as a long with a time unit suffix h-hour, m-minute, s-second, ms-millisecond (e.g. 2ms or 5s). Disabled (scanInterval=0) by default. Note: the contents of the individual files are not scanned. Rather, the fileset itself is scanned to determine if files are added or removed from the set.
```
Rewrite the description to the following wording: The scanning interval to determine whether files are added or removed from the fileset. The individual files are not scanned. The suffix for the interval of time is h-hour, m-minute, s-second, and ms-millisecond, for example, 2ms or 5s. The scanning interval is disabled by default and is disabled manually by setting the scan interval, scanInterval, to 0.

edburns replied:
d78678ec4e

chirp1 reviewed (commented):
Hi! I added my review comments for a rewrite of the wording. We can go back and forth to make changes until we come up with wording that we agree on.

edburns committed:
On branch edburns-msft-dd-1618567-fileset-scaninterval-clarify-does-not-scan-for-changes-in-file Apply changes suggested by @chirp1
modified:   dev/com.ibm.ws.kernel.metatype.helper/resources/OSGI-INF/l10n/fileset.properties

> Rewrite the description to the following wording: The scanning interval to determine whether files are added or removed from the fileset. The individual files are not scanned. The suffix for the interval of time is h-hour, m-minute, s-second, and ms-millisecond, for example, 2ms or 5s. The scanning interval is disabled by default and is disabled manually by setting the scan interval, scanInterval, to 0.

Signed-off-by: Ed Burns <edburns@microsoft.com>

edburns commented:
I assert the `CLA Needed` label is unnecessary.

This text certifies I have signed the CLA.

From: @edburns
Sent: Wednesday, September 21, 2022 2:28 PM
> EB> Subject: RE: CLA for Edward Burns from Microsoft

> EB> Hello Harry,

> EB> Thanks, now I won’t have that pesky CLA needed label on my commits.

From: @hlhoots
> HH> Subject: [EXTERNAL] RE: CLA for Edward Burns from Microsoft

> HH> Greetings Ed:

> HH> You have officially submitted the CLA for Open Liberty. Your request has been processed and you can now contribute to https://github.com/OpenLiberty/open-liberty.  

> HH> We welcome your contributions! For more details see https://github.com/OpenLiberty/open-liberty/blob/integration/README.md

edburns commented:
Thanks @LibbyBot , you are so kind and timely.

edburns commented:
However, @LibbyBot , you did miss removing the `CLA Needed` label. Perhaps your programming needs to be updated?

edburns commented:
> What is the name of the attribute that enables file scanning?

`scanInterval`.

posmith commented:
L2 message review not required

edburns commented:
Hello @chirp1 , is there anything else you need from me on this?

hiltoncv commented on dev/com.ibm.ws.kernel.metatype.helper/resources/OSGI-INF/l10n/fileset.properties:
```diff
@@ -31,4 +31,4 @@ fileset.attr.excludes.name=Excludes pattern
 #monitor
 fileset.attr.scan.name=Scan interval
-fileset.attr.scan.desc=Scanning interval to check the fileset for changes as a long with a time unit suffix h-hour, m-minute, s-second, ms-millisecond (e.g. 2ms or 5s). Disabled (scanInterval=0) by default.
+fileset.attr.scan.desc=The scanning interval to determine whether files are added or removed from the fileset. The individual files are not scanned. The suffix for the interval of time is h-hour, m-minute, s-second, and ms-millisecond, for example, 2ms or 5s. The scanning interval is disabled by default and is disabled manually by setting the scan interval, scanInterval, to 0.
```
As per the latest discussion with Karen and Daniel. We suggest the text:  
“fileset.attr.scan.desc=The scanning interval to determine whether files are added or removed from the fileset. The individual files are not scanned. The scanning interval is disabled by default and is disabled manually by setting the scan interval, scanInterval, to 0.”

chirp1 commented:
@edburns Hi! See Hilton's latest update. We communicated with Brent Daniel and came up with the new wording. Would you update the file with the new wording? Thanks!

helyarp reviewed (commented):
Changes look good!

helyarp reviewed (approved):
Reviewed and approved some time ago but adding Approve now to help close this.

mbroz2 commented:
Kicked off a PB with these changes under https://github.com/OpenLiberty/open-liberty/pull/22970
